blueprints/fastboot-test/index.js

```diff
@@ -1,9 +1,8 @@
 /* eslint-env node */
 module.exports = {
-  description: 'A test that runs against fastboot generated html',
+  description: 'Generates test config for adding custom Fastboot sanbox globals',
```

Comment:
Descriptions for `blueprints/fastboot-test/index.js` and `blueprints/fastboot-test-config/index.js` seem swapped around?

Also, there's a typo: `sanbox`.